### ISSUE TITLE
Resolve accessibility issues reported by UIUC

### ISSUE THREAD

**Issue by nwalters512** [PrairieLearn/PrairieLearn#11360]
UIUC performed another round of accessibility reviews as part of the Canvas LTI integration process. They identified several more issues that should be addressed:

- [x] There are more color contrast issues https://github.com/PrairieLearn/PrairieLearn/pull/11362:
  - For input elements, the "Invalid" text that appears on a button after the input has insufficient contrast.
  - On the question settings page, in the dropdown for the tag selector, they claim that the description text has insufficient contrast, but I wasn't able to reproduce this; according to my testing, the contrast was 4.68, which is sufficient.
  - The "GitHub access settings page" link at the bottom of the course "Staff" page has insufficient contrast.
- [x] `<pl-string-input>` has invalid `aria-labelledby` and `aria-describedby` attributes, they reference `pl-symbolic-input-*` IDs (note that that's a different element). Other input elements should be audited for similar problems. https://github.com/PrairieLearn/PrairieLearn/pull/11365
- [x] On the question settings page, when the tags or topic selector is open, the inputs with the placeholder "select a topic" or "select tags" don't have accessible labels. https://github.com/PrairieLearn/PrairieLearn/pull/11365
- [x] Some buttons don't have sufficient focus indicators. #11386 
  - On the issues page, the "Filters" and related buttons don't show focus indicators. These use a custom `.btn-med-light` style that needs the same focus ring treatment as our other buttons.
  - The user dropdown trigger in the upper-right lacks a notable focus state (the only change right now is a slight lightening of the text).
- [x] Some elements lack accessible labels https://github.com/PrairieLearn/PrairieLearn/pull/11365:
  - The yellow warning triangle that appears next to course instances / etc. with sync warnings is missing a label.
  - The "?" button on the issues page is missing a label.
- [ ] They claim that the "skip to content" link must be within a navigational landmark. I don't entirely buy this; e.g. GitHub doesn't do this. We should research this to determine if a change is actually required.
- [x] The file browser table is missing accessible table headers (they need not be visible onscreen). #11390 
- [x] ~They claim that `<h1>` must be a subset of the page's `<title>`. I'm highly skeptical of this and was unable to immediately find references to this in WCAG 2.1 AA. I'm going to do some more research and follow up with the reviewer.~ The student assessment/gradebook pages should include "Assessments" and "Gradebook" in the page's `<title>`, respectively. #11390 
- [x] The score histograms on various pages (which are SVGs) have the default role `graphics-document`. This is not well-supported. We should use `role="none"` or `role="img"` instead (determine which one is appropriate). #11390 
- [ ] They claim that the "add question" modal has no button elements. I don't believe this is true. They also claim that the fact that the form is submitted via <kbd>Enter</kbd> is a problem. I also don't think this is a valid concern; forms are by default submitted via <kbd>Enter</kbd> and I see no reason to change this.

**Comment by nwalters512** [comment]
The two "unresolved" items were resolved via discussion with the accessibility reviewer. We're all now in agreement that these are not in fact accessibility issues.
